EIPS/eip-4844.md

```diff
@@ -193,21 +193,22 @@ The resulting RLP encoding of the header is therefore:
 ```
 rlp([
     parent_hash,
-    ommers_hash,
+    0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347, # ommers hash
```

Comment:
If we're going to hardcode post-merge header values in this EIP we should maybe refer to [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675) that defines them.

Reply:
this was downstream of a request in 4895 to hardcode the values

I'm happy to add a note about the EIP, although if we want to standardize on this presentation then it will be a little silly to always reference 3675 in all future EIPs writing the header